lib/take_elevated.lua

```diff
@@ -8,6 +8,7 @@ local erl_bucket_size = tonumber(ARGV[7])
 local erl_activation_period_seconds = tonumber(ARGV[8])
 local erl_quota_amount = tonumber(ARGV[9])
 local erl_quota_expiration_epoch = tonumber(ARGV[10])
+local is_erl_enabled = ARGV[11] == "true" and true or false
```

Comment:
just in case, this is the ternary operator in lua, similar to
`condition ? true : false`